free-podcasts-screencasts-en.md

```diff
@@ -161,6 +162,9 @@
 
 * [Ember Weekend](https://emberweekend.com/episodes) (podcast)
 
+#### ml5.js
+* [Beginners Guide to Machine Learning in JavaScript](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y) (screencast)
```

Comment:
I'm not familiar with ml5.js or p5.js, so I don't know whether they should have their own categories. A helpful way to approach this question is to imagine you are someone who could benefit from this screencast. Would you be likely to be looking for an ml5 screencast, or a machine-learning screencast?

Reply:
This is not a promo or anything in case you may be thinking. I tried to add it since I find JavaScript machine learning pretty interesting stuff. But anyway, if you don't want it to be added, feel free to close the pull request. And talking about the formatting issues, I updated the previously forked repository before making changes. Not sure why it is still counting the old commits and duplicating stuff.

Reply:
Oh, I see. The addition is fine, but somehow your fork is not a clean copy of the repo's current state. I'm going to close the PR so you can start another one.

As to the appropriate category, I'd recommend "Machine Learning" or "Javascript".